package.json

```diff
@@ -29,7 +29,9 @@
     "nodemon": "^1.3.7",
     "serve-static": "^1.9.2",
     "swagger-editor": "^2.9.2",
-    "swagger-tools": "^0.8.6"
+    "swagger-tools": "^0.8.6",
+    "async": "^1.2.1",
+    "handlebars": "^3.0.3"
```

Comment:
Why handlebars is required? Isn't it a dependency of the swagger-test-templates module? Also swagger-test-templates module should be listed here